src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/haflows/HaFlowHistorySpec.groovy

```diff
@@ -0,0 +1,125 @@
+package org.openkilda.functionaltests.spec.flows.haflows
+
+import static org.openkilda.testing.Constants.WAIT_OFFSET
+
+import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.extension.failfast.Tidy
+import org.openkilda.functionaltests.helpers.HaFlowHelper
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.northbound.dto.v2.haflows.HaFlow
+
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import spock.lang.Ignore
+import spock.lang.Narrative
+import spock.lang.Shared
+
+@Narrative("""Verify that history records are created for the create/update actions.
+History record is created in case the create/update action is completed successfully.""")
+@Slf4j
+class HaFlowHistorySpec extends HealthCheckSpecification {
```

Comment:
Your `@Narrative` doesn't match class name. If you limit checks to update/create actions, your class should be named likd `HaFlowHistoryCreateAndUpdateSpec`, otherwise, you should replace `create/update actions` in your narrative with `some basic actions`.

Reply:
changed